x/stakeibc/keeper/lsm.go

```diff
@@ -1,12 +1,110 @@
 package keeper
 
 import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	errorsmod "cosmossdk.io/errors"
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	transfertypes "github.com/cosmos/ibc-go/v5/modules/apps/transfer/types"
+	channeltypes "github.com/cosmos/ibc-go/v5/modules/core/04-channel/types"
 
 	"github.com/Stride-Labs/stride/v8/x/stakeibc/types"
 )
 
-func (k Keeper) TransferLSMTokenToHost(ctx sdk.Context, lsmTokenDeposit types.LSMTokenDeposit) error {
-	// TODO [LSM]
-	return nil
+var (
+	// QUESTION: Is this too restrictive? Would we ever want to accept a LSM token from more than 1 hop away?
+	// A valid IBC path for the LSM token must only consist of 1 channel hop along a transfer channel
+	// (e.g. "transfer/channel-0")
+	IsValidIBCPath = regexp.MustCompile(fmt.Sprintf(`^%s/(%s[0-9]{1,20})$`, transfertypes.PortID, channeltypes.ChannelPrefix)).MatchString
```

Comment:
This is for the most part setup with that philosophy! For instance, the PortID and Channel prefix are both pulled from the ibc-go module so if they change in an upgrade, this will get updated automatically.

But I don't think there's a helper function for this particular check since it's a bit more restrictive than a normal IBC path (in that it can only have 1 channel-ID)

Fwiw, I think it's incredibly unlikely that this changes